### PR TITLE
LPS-118527 Check if field belongs to the valid field types

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/card-shortcut/CardShortcut.es.js [hunk 1]
```diff
@@ -30,21 +31,27 @@ export default ({fields}) => {
...
-				data-senna-off
-				href={`#${portletNamespace}card_${index}`}
-			>
-				<div className="indicator"></div>
-				<div className="field-label">{field.label}</div>
-			</a>
-		</li>
-	));
+	const shortcuts = fields.map((field, index) => {
+		if (Object.keys(fieldTypes).includes(field.type)) {
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/card-shortcut/CardShortcut.es.js:35]: Good job, @alinedoleron! 😄 Elegant and simple fix. 🙂

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**